SplunkforPaloAltoNetworks/default/data/ui/views/cortex_xdr_incidents.xml

```diff
@@ -0,0 +1,319 @@
+<form>
+  <label>Cortex XDR Incidents</label>
+  <search id="basesearch">
+    <query>sourcetype="pan:xdr_incident" $incident_id$ $severity$ $status$ | dedup incident_id | eval assigned_user_pretty_name=replace(assigned_user_pretty_name,"null", "UNASSIGNED") | eval score=coalesce(if(manual_score="null", null(), manual_score), if(rule_based_score="null", null(), rule_based_score))  | eval "Last Updated" = strftime(_time, "%Y-%d-%m %H:%M:%S") | spath  path="incident_sources{}" output="Incident Sources" | spath  path="hosts{}" output="dest_hosts" | table "Last Updated", starred, incident_id, description, assigned_user_pretty_name, severity, dest_hosts, score, status, "Incident Sources", xdr_url, "Open in XDR" </query>
```

Comment:
1. You're coalsceing some of these fields in props so shouldn't need to do it here.

2. Don't use dedup, it's too slow.

3. You're doing a lot in the base search that doesn't apply to all panels. Can you break this out a bit?

Reply:
Optimized the base search and dashboard.